codegenerator/cli/npm/envio/src/Internal.res

```diff
@@ -3,7 +3,7 @@ type eventBlock
 type eventTransaction
 
 @genType
-type genericEvent<'params, 'transaction, 'block> = {
+type genericEvent<'params, 'block, 'transaction> = {
```

Comment:
Was this a bug? Makes me nervous swapping params when we do all this type casting 😅

Reply:
I've created the type in the previous PR, but didn't use it. In the PR I've decided to have arguments in different order. This should be safe, since I also added tests for types.